cleanlab_studio/utils/data_enrichment/enrich.py

```diff
@@ -15,14 +14,14 @@
 
 def enrich_data(
     studio: Studio,
-    prompt: str,
     data: pd.DataFrame,
+    prompt: str,
     *,
-    regex_replacements: Optional[Union[Replacement, List[Replacement]]] = None,
-    return_values: Optional[List[str]] = None,
+    replacements: Optional[Union[Replacement, List[Replacement]]] = None,
+    constrain_outputs: Optional[List[str]] = None,
     optimize_prompt: bool = True,
     subset_indices: Optional[Union[Tuple[int, int], List[int]]] = (0, 3),
-    metadata_column_name: str = "metadata",
+    new_column_name: str = "metadata",
```

Comment:
This is marked as optional in the API spec, will keep it optional here as well.
I can see how it could be nice (esp for new users randomly testing this out) to not have to worry about specifying a column name (less decisions lol)